src/devices/machine/pccard.h

```diff
@@ -18,6 +19,8 @@ class device_pccard_interface : public device_interface
 
 protected:
 	device_pccard_interface(const machine_config &mconfig, device_t &device);
+
+	pccard_slot_device *m_slot;
```

Comment:
Is there a reason this isn’t a `const` pointer and set in the initialiser list?  Device parents can’t change after construction.

Reply:
I've changed it to be set inside the initialiser list, but I don't see how I can make it `const`. No bus devices I've looked at seem to have it `const`.

Reply:
Here’s one where it’s `const`: https://github.com/mamedev/mame/blob/mame0252/src/devices/bus/coco/cococart.h#L222

Here’s another: https://github.com/mamedev/mame/blob/mame0252/src/devices/bus/sms_ctrl/smsctrl.h#L123

And another: https://github.com/mamedev/mame/blob/mame0252/src/devices/bus/vboy/slot.h#L161